daal4py/mb/model_builders.py

```diff
@@ -283,8 +283,13 @@ def predict_proba(self, X):
 
 
 def convert_model(model):
-    gbm = GBTDAALModel()
-    gbm._convert_model(model)
+    try:
```

Comment:
Could you please describe, why do you need this try-except block?

Reply:
Our current version of dispatching doesn't account for all scenarios. We trying to support scikit-style model objects, but also native types from XGBoost, LightGBM, CatBoost. In some cases we are creating a `GBTDAALBase` object when we actually expect a `GBTDAALRegressor` or `GBTDAALClassifier`.
The solution in this PR is to "ask for forgiveness, not permission", but I have created a follow-up task to consolidate the dispatching.
For now, this is an improvement, as it increases the compatibility of the `convert_model()` function to more types of input model types.